app/cdap/components/DataPrep/DataPrepSidePanel/SamplingTab/SamplingTab.scss

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+.sampling-tab {
```

Comment:
We're doing CSS-in-JS for new code. `styled-components` is best - see https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/components/Metadata/Lineage/LineageDiagram/styles.tsx for an example.